homeassistant/components/template/fan.py

```diff
@@ -147,7 +147,7 @@ def __init__(
         self._preset_mode_template = config.get(CONF_PRESET_MODE_TEMPLATE)
         self._oscillating_template = config.get(CONF_OSCILLATING_TEMPLATE)
         self._direction_template = config.get(CONF_DIRECTION_TEMPLATE)
-        self._supported_features = 0
+        self._attr_supported_features = 0
```

Comment:
Lets move this into the class variables

Reply:
Done... it's already the default in `FanEntity`